meta-mender-core/classes/mender-sdimg.bbclass

```diff
@@ -114,8 +114,6 @@ IMAGE_CMD_sdimg() {
                           $boot_env_size_mb)
     CALC_ROOTFS_SIZE=$(expr $REMAINING_SIZE / 2)
 
-    MENDER_PARTITION_ALIGNMENT_KB=$(expr ${MENDER_PARTITION_ALIGNMENT_MB} \* 1024)
-
```

Comment:
Careful! The reference below is a shell reference. You need to add the `{}` to it.